Cargo.toml

```diff
@@ -204,7 +206,7 @@ ismp = { default-features = false, version = "0.2.2" }
 pallet-ismp = { default-features = false, version = "15.0.0" }
 pallet-ismp-rpc = { default-features = false, version = "15.0.0" }
 pallet-ismp-runtime-api = { default-features = false, version = "15.0.0" }
-ismp-grandpa = { default-features = false, version = "15.0.0" }
+ismp-grandpa = { default-features = false, version = "15.0.1" }
```

Comment:
they released 16.0.0 version for this pallet, but I guess we must update to the corresponding Polkadot version (polkadot-stable2409)... let's remember to update these dependencies when the time come

Reply:
I guess it should not simple to forget it ... Quite sure that otherwise it'll not compile :smile: